### PR TITLE
Confirm Password Field Added

### LINKED ISSUE #344: Add re-enter password field in forget password page.
## Describe the bug
Currently the forget password page do not have field to confirm the new password. 

## To Reproduce
1.  headover to forget password page

## Expected behavior
There should be a confirm password field so as to make sure password is correctly entered.

## Logs
image

## Environment (please complete the following information):
 - OS: all
 - Browser: all

## Additional context
also change login button text from "login"  -> "reset"

### PR DESCRIPTION
**Pull request by Garima-7**
Fixed #344

#### Describe the changes you've made
Changed the file "forgetPass.html"
Added the Confirm Password Field to it.
Changed the text of button to reset.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Screenshot of added functionality
pic

#### Checklist

- [x] I have mentioned related issue number in PR description.
- [x] I have added screenshots and related demo links of my changes..
- [x] The title of my pull request is a short description of the requested changes.
- [ ] I have added my name in the contributors list at the end of README.md file.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)

### CODE REVIEW

File: forgetPass.html [hunk 1]
```diff
@@ -16,145 +17,144 @@
...
 
-      </div>
+            </div>
 
 
-      <div>
-        <input type="checkbox" class="checkbox" id="chk" />
-        <label class="label" for="chk">
+            <div>
+                <input type="checkbox" class="checkbox" id="chk" />
```
Comment (nlok5923) [inline forgetPass.html:136]: Please revert unnecessary changes

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks PTAL

File: forgetPass.html (hunk 1 above)
Comment (Garima-7) [reply forgetPass.html]: Sorry for that.
OK bhaiya I will resolve it.
I am little weak in git commands.
